Bump env XDR

dmkozh opened the pull request:
### What

Bump env XDR

### Why

Catch up to the XDR changes

### Known limitations

N/A

graydon commented:
@dmkozh I figured I'd try to land this once I fixed the CI breakage and my PRs landed  -- I just rebased your change on main and rebuilt the wasms. hope you don't mind!

dmkozh commented:
> @dmkozh I figured I'd try to land this once I fixed the CI breakage and my PRs landed -- I just rebased your change on main and rebuilt the wasms. hope you don't mind!

Thanks for helping with this!
